pontoon/base/static/js/translate.js

```diff
@@ -1566,6 +1566,19 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Prevent too long translation submissions
+     */
+    translationLengthLimitExceeded: function (translation) {
+      if (this.translationLengthLimit && this.stripHTML(translation).length > this.translationLengthLimit) {
+        this.endLoader('Translation too long.', 'error');
```

Comment:
We use "Translation deleted", "Translation saved", "Translation approved", so I think it's OK to use "Translation too long" and follow this pattern.

I agree about the second part, let's remove the notifications. Besides, there are other pieces that are already duplicated the calling the function (if clause and return).